<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pqr_GroupBox</class>
 <widget class="QGroupBox" name="pqr_GroupBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1" colspan="2">
      <widget class="QComboBox" name="pqr_method_comboBox"/>
     </item>
     <item row="3" column="0" alignment="Qt::AlignRight|Qt::AlignVCenter">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Output Molecule Object:</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="QStackedWidget" name="pqr_method_stackedWidget">
       <widget class="QWidget" name="page_pdb2pqr">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="1" column="0" alignment="Qt::AlignRight|Qt::AlignVCenter">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Command line flags:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="2" alignment="Qt::AlignLeft|Qt::AlignTop">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>pdb2pqr adds hydrogens and missing sidechain atoms and assigns partial charges and radii.
It removes ligands and modified residues.</string>
             </property>
             <property name="textFormat">
              <enum>Qt::PlainText</enum>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QCheckBox" name="pdb2pqr_warnings_checkBox">
             <property name="text">
              <string>Ignore warnings</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" alignment="Qt::AlignVCenter">
            <widget class="QLineEdit" name="pdb2pqr_flags_lineEdit"/>
           </item>
           <item row="3" column="0">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_pymol">
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0" alignment="Qt::AlignLeft|Qt::AlignTop">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>PyMOL can directly generate PQR files using standard protein residues and AMBER charges.</string>
             </property>
             <property name="textFormat">
              <enum>Qt::PlainText</enum>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="3" column="1" colspan="2" alignment="Qt::AlignVCenter">
      <widget class="QLineEdit" name="pqr_output_mol_lineEdit"/>
     </item>
     <item row="0" column="0" alignment="Qt::AlignVCenter">
      <widget class="QCheckBox" name="pqr_prepare_mol_checkBox">
       <property name="text">
        <string>Prepare Molecule</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0" alignment="Qt::AlignRight|Qt::AlignVCenter">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Method:</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
